restful-ws/4.0/_index.md

```diff
@@ -1,32 +1,38 @@
 ---
 title: "Jakarta RESTful Web Services 4.0 (under development)"
 date: 2022-04-15
-summary: "Release for Jakarta EE 10"
+summary: "Release for Jakarta EE 11"
 ---
+
 Jakarta RESTful Web Services provides a foundational API to develop web services following the Representational
 State Transfer (REST) architectural pattern.
 
 The goal of this release is to provide better alignment between Jakarta RESTful Web Services and 
 Jakarta Contexts Dependency Injection (CDI). This will be the first _backward incompatible_ release of 
 Jakarta REST.  
 
-* Drop support for @Context injection and related artifacts - [Issue #951](https://github.com/eclipse-ee4j/jaxrs-api/issues/951)
+### New features, enhancements or additions
 * Define CDI scopes for all Jakarta REST types - [Issue #952](https://github.com/eclipse-ee4j/jaxrs-api/issues/952)
```

Comment:
https://github.com/eclipse-ee4j/jaxrs-api/issues/952 should be https://github.com/jakartaee/rest/issues/952